modules/data.land/inst/build_soil_texture_variables.R

```diff
@@ -0,0 +1,159 @@
+)#end list
+stext.polygon[[ 5]] = list(sand = c(0.525,0.450,0.225,0.425,0.525)
+                           ,clay = c(0.200,0.275,0.275,0.075,0.075)
+)#end list
+stext.polygon[[ 6]] = list(sand = c(0.800,0.650,0.450,0.450,0.525)
+                           ,clay = c(0.200,0.350,0.350,0.275,0.200)
+)#end list
+stext.polygon[[ 7]] = list(sand = c(0.200,0.000,0.000,0.200)
+                           ,clay = c(0.400,0.400,0.275,0.275)
+)#end list
+stext.polygon[[ 8]] = list(sand = c(0.450,0.200,0.200,0.450)
+                           ,clay = c(0.400,0.400,0.275,0.275)
+)#end list
+stext.polygon[[ 9]] = list(sand = c(0.650,0.450,0.450)
+                           ,clay = c(0.350,0.550,0.350)
+)#end list
+stext.polygon[[10]] = list(sand = c(0.200,0.000,0.000)
+                           ,clay = c(0.400,0.600,0.400)
+)#end list
+stext.polygon[[11]] = list(sand = c(0.300,0.300,0.000)
+                           ,clay = c(0.400,0.700,0.700)
+)#end list
+stext.polygon[[12]] = list(sand = c(NA,NA)
+                           ,clay = c(NA,NA)
+)#end list
+stext.polygon[[13]] = list(sand = c(NA,NA)
+                           ,clay = c(NA,NA)
+)#end list
+stext.polygon[[14]] = list(sand = c(0.200,0.075,0.000,0.000)
+                           ,clay = c(0.000,0.125,0.125,0.000)
+)#end list
+stext.polygon[[15]] = list(sand = c(0.300,0.000,0.000)
+                           ,clay = c(0.700,1.000,0.700)
+)#end list
+stext.polygon[[16]] = list(sand = c(0.450,0.300,0.300,0.450)
+                           ,clay = c(0.550,0.700,0.400,0.400)
+)#end list
+stext.polygon[[17]] = list(sand = c(0.300,0.000,0.000,0.200)
+                           ,clay = c(0.400,0.700,0.600,0.400)
+)#end list
+nstext.polygon      = length(stext.polygon)
+
+for(n in 1:nstext.polygon){
+  sand.now = stext.polygon[[n]]$sand
+  clay.now = stext.polygon[[n]]$clay
+  stext.polygon[[n]]$silt = pmax(0,pmin(1,1.-sand.now-clay.now))
+}#end for
+stext.polygon  <<- stext.polygon
+nstext.polygon <<- nstext.polygon
+#==========================================================================================#
+#==========================================================================================#
+
+#----- Define some prescribed fractions. -----------------------------------------------#
+xsand.def = c( 0.920, 0.825, 0.660, 0.200, 0.410, 0.590
+               , 0.100, 0.320, 0.520, 0.060, 0.200, 0.200
+               , 0.333, 0.075, 0.100, 0.375, 0.125)
+xclay.def = c( 0.030, 0.060, 0.110, 0.160, 0.170, 0.270
+               , 0.340, 0.340, 0.420, 0.470, 0.600, 0.200
+               , 0.333, 0.050, 0.800, 0.525, 0.525)
+
+soil.name = c("Sand","Loamy sand","Sandy loam","Silt loam","Loam","Sandy clay loam"
+              ,"Silty clay loam","Clayey loam","Sandy clay","Silty clay","Clay"
+              ,"Peat","Bedrock","Silt","Heavy clay","Clayey sand","Clayey silt")
+
+texture <- read.csv("~/pecan/modules/data.land/data/texture.csv",header=TRUE,stringsAsFactors = FALSE)
```

Comment:
It's not intended to be run by users, it's used to build an RData file of important look-up tables, the latter of which IS accessed by system.file. This script only needs to be rerun if that RData file needs to be regenerated.